### ISSUE TITLE
Should `summary` be included?

### ISSUE THREAD

**Issue by mxmason** [KittyGiraudel/focusable-selectors#8]
As far as I know, `details/summary` is moderately popular. Authors who use this library on nodes with `summary` in them might see some weird behaviors.

**Comment by KittyGiraudel** [comment]
The difficulty with `summary` is that only **the first** `summary` element within a `detail` element should be considered focusable; any other should not. **And** a `summary` element outside of a `detail` element should not be focusable.

I mean, maybe we’re over-thinking it and we just consider all `summary` focusable for sake of simplicity? But it would be technically incorrect.

Or maybe now that the `:has()` selector has good support:
```js
detail:has(summary) summary:first-of-type
```

**Comment by mxmason** [comment]
Is it spec-compliant to have >1 `summary` in `details`? How does the browser even handle that? 🧐 I’d check if I weren’t on my phone.

If > 1 summary is allowed, I think we could get the right behavior without :has():

```
details > summary:first-of-type
```

No?

Edit: Silly me, this won’t work if we’re checking a node directly, sorry. I would argue in favor of simply considering all summary elements focusable.

**Comment by KittyGiraudel** [comment]
I’m not entirely sure. Similarly, although a `summary` should be a direct descendant of a `detail` element, I think browsers might support having a `div` in between (or other element) for styling purposes, which means the `>` might be riskY?

**Comment by KittyGiraudel** [comment]
Yes, I guess we can just add `summary`. Probably good enough? I’d argue better considering too many elements focusable than not enough. I’ll issue a commit.

**Comment by KittyGiraudel** [comment]
Addressed in #9.
